docs/ReleaseNotes.md

```diff
@@ -66,7 +66,7 @@ The `RecordStoreState` constructors have been deprecated or marked [`INTERNAL`](
 // begin next release
 ### NEXT_RELEASE
 
-* **Bug fix** Fix 1 [(Issue #NNN)](https://github.com/FoundationDB/fdb-record-layer/issues/NNN)
+* **Bug fix** `FDBRecordStore.checkRebuildIndexes` uses the count of records to be scanned for the new indexes to determine overall emptiness [(Issue #577)](https://github.com/FoundationDB/fdb-record-layer/issues/577)
```

Comment:
Something like:

```suggestion
* **Bug fix** `FDBRecordStore.checkRebuildIndexes` now uses the count of all records to determine overall emptiness [(Issue #577)](https://github.com/FoundationDB/fdb-record-layer/issues/577)
```

Just to make it a description of the fix rather than the bug. (That is, unless the fix is changed to just remove the `omit_unsplit_suffix` removal.)

Reply:
Accepted. Please feel free to squash when we get to merging.